docs/reference/filebeat/running-on-kubernetes.md

```diff
@@ -164,17 +165,19 @@ filebeat   32        32        0         32           0           <none>
 Log events should start flowing to Elasticsearch. The events are annotated with metadata added by the [add_kubernetes_metadata](/reference/filebeat/add-kubernetes-metadata.md) processor.
 
 
-## Parsing json logs [_parsing_json_logs]
+## Parsing JSON logs [_parsing_json_logs]
+
+The application logs from workloads running on Kubernetes are usually in JSON format. In such cases, special handling can be applied to parse the JSON logs properly and decode them into fields. 
 
-It is common case when collecting logs from workloads running on Kubernetes that these applications are logging in json format. In these case, special handling can be applied so as to parse these json logs properly and decode them into fields. Bellow there are provided 2 different ways of configuring [filebeat’s autodiscover](/reference/filebeat/configuration-autodiscover.md) so as to identify and parse json logs. We will use an example of one Pod with 2 containers where only one of these logs in json format.
+You can configure [Filebeat autodiscover](/reference/filebeat/configuration-autodiscover.md) to identify and parse JSON logs in two different ways. We will use an example of one pod with two containers where only the logs of one container are in JSON format.
 
 Example log:
 
 ```
 {"type":"log","@timestamp":"2020-11-16T14:30:13+00:00","tags":["warning","plugins","licensing"],"pid":7,"message":"License information could not be obtained from Elasticsearch due to Error: No Living connections error"}
 ```
 
-1. Using `json.*` options with templates
+- Using `json.*` options with templates:
```

Comment:
I'm not sure an unordered list is the right solution here either. I don't think it's clear enough that these are different methods of accomplishing the same task. You could instead turn these into headers. That would also allow you to remove the extra indentation.